contracts/sfc/Staker.sol

```diff
@@ -369,13 +406,13 @@ contract Stakers is StakersConstants {
 
         require(delegations[delegator].amount != 0, "delegation doesn't exist");
         require(delegations[delegator].deactivatedTime == 0, "delegation is deactivated");
+        require(checkBonded(), "not passed enough time since stake is bonded");
```

Comment:
Won't this immediately be true?

function checkBonded() internal returns (bool) {
	maintainCapReachedDate(); // capReachedDate would be 0
	return rewardsAllowed(); //block.timestamp.sub(0) >= 6 months == true
}

function maintainCapReachedDate() internal {
	uint256 current = bondedRatio();
	uint256 target = bondedTargetRewardUnlock();
	if (current >= target && capReachedDate == 0) {
		changeCapReachedDate(block.timestamp);
	} else if (current < target && capReachedDate != 0) {
		changeCapReachedDate(0);
	}
}
function rewardsAllowed() public view returns (bool) {
	return block.timestamp.sub(capReachedDate) >= unbondingUnlockPeriod();
}

function unbondingUnlockPeriod() public pure returns (uint256) {
  return 60 * 60 * 24 * 30 * 6; // 6 months
}

Reply:
Should be fixed in ded1f86, plesae check